### DISCUSSION TITLE
Nesting and Luma key in 0.2

### DISCUSSION THREAD

**Discussion by nafergo** [olive-editor/olive#1833] [Q&A]
I'm using 0.1 because I can't do sequence nesting and Luma key in 0.2. I want to create a couple of motion titles. Any ideas or suggestions?

**Comment by GUY-BBS** [comment]
Motion titles can be done using a combination of nodes and key-framing the appropriate Transform parameters.

The clips below show an animated title strap made in 0.2.

https://user-images.githubusercontent.com/49677111/149625662-98e244e2-5fa2-4b49-8420-6e0b627f1ad5.mp4


https://user-images.githubusercontent.com/49677111/149625665-234712e1-671c-4c6f-bc7b-6e5d9ec0434c.mp4

While at present there is no Luma Key as such in 0.2 there are a couple of work-arounds.

If you could indicate how and what you wish to key, perhaps I can suggest something.
